### PR TITLE
95m weather service

### PR DESCRIPTION
**Pull request by happy25**
Make it rain!!!!  Get it?

### CODE REVIEW

File: StudentDriver/StudentDriver/Models/User.cs [hunk 1]
```diff
@@ -12,7 +12,7 @@ public class User
         public string Email { get; set; }
         public string FirstName { get; set; }
         public string LastName { get; set; }
-        public int DrivingSchoolId { get; set; }
+        public string DrivingSchoolId { get; set; }
```
Comment (sorianog) [inline StudentDriver/StudentDriver/Models/User.cs:15]: Uh-oh, looks like your id is a string. But the server's id is a Number: 
````javascript 
schoolId: Number
````

What should we conform too? @DylanWalseth @dobber1611

File: StudentDriver/StudentDriver/OAuth/OAuthController.cs [hunk 2]
```diff
@@ -11,13 +12,18 @@ public class OAuthController
...
+        public async Task<Response> MakePostRequest(string url, JObject jsonJObject)
+        {
+            return await MakeOAuthRequest(OAuthRequest.Post, url, AccountHandler.GetSavedFacebookAccount(), null, jsonJObject.ToString());
         }
 
         public async Task<string> VerifyAccount(string url, Account account)
         {
             if (account == null) return null;
-            var response = await MakeOAuthRequest("POST", url, account);
+            var response = await MakeOAuthRequest(OAuthRequest.Post, url, account);
```
Comment (sorianog) [inline StudentDriver/StudentDriver/OAuth/OAuthController.cs:26]: Ah, an authorized POST request. Good.

File: StudentDriver/StudentDriver/Models/User.cs (hunk 1 above)
Comment (zawadzkip) [reply StudentDriver/StudentDriver/Models/User.cs:15]: Are we using unique id's? I thought we were using mongo id's for the time being?

**Review by zawadzkip — APPROVED** [review]
👍

**Pull request merged**